client/prisma/schema.prisma

```diff
@@ -34,3 +34,13 @@ model Listener {
 
     @@map("listener")
 }
+
+model Lockup {
+  id       Int @default(autoincrement()) @id
+  address  String @unique
```

Comment:
There might be more than 1 lockup per account

Reply:
Yea for some additional context here in case you weren't aware @tomhirst, @DanielVF is in the process of reworking on the lockup contract and it should support multiple lockups of different lengths. Not merged yet though of course, so I think we can merge this because it is currently correct and do the changes required for multiple lockups later as there'll be more than just this PR that needs updating.

Reply:
Another thought: currently you can still modify your existing lockup. It might make sense to add another boolean to the table e.g. `active` or `latest` that would indicate which is the last lockup user has created. This would enable us to (instead of modify existing lockup in the table to) have data for historical lockups on hand in case we would ever want to do any analysis based on that.

Reply:
> Yea for some additional context here in case you weren't aware @tomhirst, @DanielVF is in the process of reworking on the lockup contract and it should support multiple lockups of different lengths. Not merged yet though of course, so I think we can merge this because it is currently correct and do the changes required for multiple lockups later as there'll be more than just this PR that needs updating.

Thanks, @tomlinton, got it. Yeah, we can merge this and then open a new issue to refactor the UI to match the contract changes. I can take ownership of that one when @DanielVF's work is merged.

> Another thought: currently you can still modify your existing lockup. It might make sense to add another boolean to the table e.g. `active` or `latest` that would indicate which is the last lockup user has created. This would enable us to (instead of modify existing lockup in the table to) have data for historical lockups on hand in case we would ever want to do any analysis based on that.

Smart, I like it. Would be nice to build a dashboard for users displaying this historic data.

Have extended this up as you suggested.

Reply:
sweet thanks 👍